modules/images/webp-uploads/load.php

```diff
@@ -96,6 +96,34 @@ function webp_uploads_create_sources_property( array $metadata, $attachment_id )
 			continue;
 		}
 
+		$original_image_filesize = isset( $metadata['filesize'] ) ? (int) $metadata['filesize'] : 0;
+		$webp_image_filesize     = isset( $image['filesize'] ) ? (int) $image['filesize'] : 0;
+		if ( $original_image_filesize <= 0 && $webp_image_filesize <= 0 ) {
+			continue;
+		}
+
+		/**
+		 * Filter whether to prioritize the file type over the mime type.
+		 *
+		 * By default the performance lab plugin will use the mime type with the smaller filesize
+		 * rather than defaulting to `webp`.
+		 *
+		 * @since n.e.x.t
+		 *
+		 * @param bool $preferred_filesize Prioritize file size over mime type. Default true.
+		 */
+		if (
+			apply_filters( 'webp_prioritise_filesize_over_mime_type', true )
```

Comment:
for filter name, the plugin uses the module name prefix. Also, lets stick to the US spelling of "prioritize" with a "z". So maybe `webp_uploads_prioritize_filesize_over_mime_type`

Reply:
Thanks for the feedback. I have updated the filter name to `webp_uploads_discard_larger_generated_images` as you suggest in other comments. That makes more sense.